kolibri/core/assets/src/api-resource.js

```diff
@@ -915,6 +916,16 @@ export class Resource {
     return this.accessEndpoint('POST', listName, params);
   }
 
+  /**
+   * Call a POST on a custom list endpoint
```

Comment:
I'd just change this test to something like:
   * Call a POST on a custom list endpoint forcing the use of
   * 'multipart/form-data' as Mimetype instead of the common 'application/json'